packages/admin/cms-admin/src/dam/DataGrid/FolderDataGrid.tsx

```diff
@@ -380,11 +380,19 @@ const FolderDataGrid = ({
             field: "importSourceType",
             headerName: intl.formatMessage({
                 id: "comet.dam.file.importSourceType",
-                defaultMessage: "Source",
+                defaultMessage: "Type/Format",
             }),
+            headerAlign: "left",
+            align: "left",
+            minWidth: 140,
             renderCell: ({ row }) => {
-                if (isFile(row) && row.importSourceType && importSources?.[row.importSourceType]) {
-                    return importSources[row.importSourceType].label;
+                if (isFile(row) && row.mimetype) {
+                    return row.mimetype;
+                } else if (isFolder(row)) {
+                    return intl.formatMessage({
+                        id: "comet.dam.file.type.folder",
+                        defaultMessage: "Folder",
+                    });
```

Comment:
This feature was added last week (#1756), please don't remove it

Reply:
I have added a new column and left the old one as is